specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/playbookConfigurations.json

```diff
@@ -0,0 +1,511 @@
+          "description": "The AAD object ID of the entity that last updated the playbook configuration."
+        }
+      }
+    },
+    "PlaybookConfigurationSource": {
+      "type": "object",
+      "description": "The source event types which evaluate the playbook configuration set of rules. For example - security alerts and recommended tasks.",
+      "properties": {
+        "eventSource": {
+          "type": "string",
+          "description": "A valid event source type."
+        },
+        "ruleSets": {
+          "type": "array",
+          "description": "A set of rules which evaluate upon event interception.",
+          "items": {
+            "$ref": "#/definitions/PlaybookConfigurationRuleSet"
+          }
+        }
+      }
+    },
+    "PlaybookConfigurationScope": {
+      "type": "object",
+      "description": "The subscription's resources scope",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The resources scope description."
+        },
+        "scopePath": {
+          "type": "string",
+          "description": "The resources scope path."
+        }
+      }
+    },
+    "PlaybookConfigurationAction": {
+      "type": "object",
+      "description": "The action that should be triggered.",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The triggered resource id."
+        },
+        "resourceProviderNamespace": {
+          "type": "string",
+          "description": "The resource provider name."
+        },
+        "uri": {
+          "type": "string",
+          "description": "The uri that should be triggered by an Http GET request.",
+          "format": "uri"
+        }
+      }
+    },
+    "PlaybookConfigurationRuleSet": {
+      "type": "object",
+      "description": "A rule set which evaluates all its rules upon an event interception.",
+      "properties": {
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PlaybookConfigurationTriggeringRule"
+          }
+        }
+      }
+    },
+    "PlaybookConfigurationTriggeringRule": {
+      "type": "object",
+      "description": "A logic rule which evaluates upon event interception. The rule is configured by comparing the specified expected value, in an expected property within the event model, by the specified operator.",
+      "properties": {
+        "propertyJPath": {
+          "type": "string",
+          "description": "The JPath of the entity model property that should be checked."
+        },
+        "propertyType": {
+          "type": "string",
+          "description": "The data type of the compared operands.",
+          "enum": [
+            "System.String",
+            "System.Boolean",
+            "System.Byte",
+            "System.Int16",
+            "System.Int32",
+            "System.Single",
+            "System.Double",
+            "System.Char"
+
+          ]
```

Comment:
The list of types is at odds with `propertyJPath`. By looking at this API, my expectation is that the JPath will be executed on something and the propertyType is expected to match. JSON supports the following data types: string, number, object, array, boolean, null. What will be the differences between byte, int16, int32, and double?

Reply:
Done. Good call! And agreed. Changed this to support the standard Swagger/JSON data types:
"string", "integer", "number", "boolean"